### PR TITLE
add button

### PR DESCRIPTION
**Pull request by Fme82**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,7 @@
 </head>
 <body>
     <h1>Git Demo</h1>
+    <button id = "boton">Click here!</button>
```
Comment (rivannadev) [inline index.html:11]: I see

File: main.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+var boton = document.getElementById ('boton');
+boton.addEventListener('click', getAlert);
```
Comment (rivannadev) [inline main.js:2]: event listener...

**Review by rivannadev — APPROVED** [review]

File: main.js [hunk 3]
```diff
@@ -2,5 +2,5 @@ var boton = document.getElementById ('boton');
 boton.addEventListener('click', getAlert);
 
 function getAlert () {
-    alert ('Hola');
+    alert ('Hello');
```
Comment (rivannadev) [inline main.js:5]: I need the comment in russian.

Comment (Fme82) [reply main.js:5]: Resuleto

**Pull request merged**